src/components/side-panel/panel-header.js

```diff
@@ -273,6 +277,14 @@ function PanelHeaderFactory() {
       );
     }
   }
+  return connect(undefined, {
```

Comment:
you are adding tracking to `kepler.gl/src` directly here. We don't want to do that. Also, we should now randomly call connect in just this component. connect should only called once in the top level component. If you need, we can add a new `ui-state` action `TOGGLE_SIDE_PANEL_HEADER` and `showHeaderDropdown` prop in `ui-state` to trigger this behavior